log4j-api/src/main/java/org/apache/logging/log4j/util/PropertySource.java

```diff
@@ -34,7 +38,7 @@
 
     /**
      * Returns the order in which this PropertySource has priority. A higher value means that the source will be
-     * applied later so as to take precedence over other property sources.
+     * searched later and can be overridden by other property sources.
```

Comment:
The documentation actually says the opposite: sources with lower numerical priority override the others.